pkg/sql/sem/builtins/generator_builtins.go

```diff
@@ -2065,13 +2065,13 @@ const spanKeyIteratorChunkBytes = 8 << 20 // 8MiB
 var rangeKeyIteratorType = types.MakeLabeledTuple(
 	// TODO(rohany): These could be bytes if we don't want to display the
 	//  prettified versions of the key and value.
-	[]*types.T{types.String, types.String},
-	[]string{"key", "value"},
+	[]*types.T{types.String, types.String, types.String},
```

Comment:
I wonder: Should we make this new column types.Decimal? That is the type of crdb_internal_mvcc_timestamp column is.  I could go either way..

Reply:
In https://github.com/cockroachdb/cockroach/pull/90848/files#diff-ed2b0d994e60d34e9f853343fd43ee6e14ca234027786774a143e5b73b5a2698R111 im currently hashing the hlc.Timestamp.String() representation of an MVCC timestamp. As discussed offline I wanted uniformity between the output of this builtin and what we hash so that in the future we can write statements such as:

```
WITH sfp AS (SELECT xor_agg(fnv64(key, ts::BYTES, value)) AS scanfp from crdb_internal.scan(crdb_internal.table_span('t'::regclass::int)))
  SELECT count(*) FROM crdb_internal.fingerprint(crdb_internal.table_span('t'::regclass::int),'$before_insert'::TIMESTAMPTZ, false) AS efp, sfp WHERE efp = sfp.scanfp;
```

to test that our fingerprinting is working as expected. We could go either way here:
1. hash the decimal representation of the timestamp and change scan to return a decimal instead
2. hash the string representation of the timestamp and keep hashing behaviour what it is at the moment

In the interest of moving things ahead, I'll leave this as a string for now but while reviewing #90848 if we decide decimal is a better representation in both places, I'll make the switch there.